src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,11 +1,26 @@
 package core.mate.academy.model;
 
+import java.util.Random;
+
 /**
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private static final int MAX_PRODUCTIVITY = 1000;
+    private int productivity;
+
     public Bulldozer() {
+        Random rand = new Random();
+        productivity = rand.nextInt(MAX_PRODUCTIVITY);
```

Comment:
better not to use random fields in default constructor, for developer that may use your code it will confusing behaviour. 
Default constructor creates empty objects

Reply:
done